Slack.Automation/Promact.Core.Repository/TaskMailRepository/TaskMailRepository.cs

```diff
@@ -359,804 +382,439 @@ public TaskMailRepository(IRepository<TaskMail> taskMail, IStringConstantReposit
         }
 
         /// <summary>
-        /// Method to generate template body
-        /// </summary>
-        /// <param name="leaveRequest">TaskMail template object</param>
-        /// <returns>template emailBody as string</returns>
-        private string EmailServiceTemplateTaskMail(List<TaskMailDetails> taskMail)
-        {
-            Erp.Util.Email_Templates.TaskMail leaveTemplate = new Erp.Util.Email_Templates.TaskMail();
-            // Assigning Value in template page
-            leaveTemplate.Session = new Dictionary<string, object>
-            {
-                {_stringConstant.TaskMailDescription, taskMail},
-            };
-            leaveTemplate.Initialize();
-            var emailBody = leaveTemplate.TransformText();
-            return emailBody;
-        }
-
-        /// <summary>
-        ///Method geting list of Employee 
+        ///Method geting Employee or list of Employees 
         /// </summary>
         /// <param name="userId"></param>
         /// <returns></returns>
-        public async Task<List<TaskMailUserAc>> GetAllEmployee(string userId)
+        public async Task<List<TaskMailReportAc>> GetAllEmployeeAsync(string userId)
```

Comment:
This method has a lot of repeated code which can be extracted to separate method for better managability